test/unit/core.unit.test.ts

```diff
@@ -9,38 +10,173 @@ import {
   getMangroveVersionDeployments,
   getMgvReaderVersionDeployments,
   getMgvOracleVersionDeployments,
+  getAllMangroveVersionDeploymentsPerNetwork,
+  getLatestMangrovePerNetwork,
+  getAllMgvReaderVersionDeploymentsPerNetwork,
+  getLatestMgvReaderPerNetwork,
+  getAllMgvOracleVersionDeploymentsPerNetwork,
+  getLatestMgvOraclePerNetwork,
+  getLatestCoreContractsPerNetwork,
 } from "../../src/core";
+import { firstVersionDeploymentsToVersionNetworkDeployment } from "./unitTestUtil";
 
 describe("core.ts", () => {
-  describe("getMangroveVersionDeployments", () => {
-    it("should find the latest deployment first", () => {
-      const result = getMangroveVersionDeployments({ released: undefined });
-      assert.equal(result, Mangrove_v2_0_1);
-      // NB: Add older old versions here
-      [].forEach((version) => {
-        assert.notEqual(result, version);
+  describe("Mangrove contract", () => {
+    describe("getMangroveVersionDeployments", () => {
+      it("should find the latest deployment first", () => {
+        const result = getMangroveVersionDeployments({ released: undefined });
+        assert.equal(result, Mangrove_v2_0_1);
+        // NB: Add older old versions here
+        [].forEach((version) => {
+          assert.notEqual(result, version);
+        });
+      });
+    });
+
+    describe("getAllMangroveVersionDeploymentsPerNetwork", () => {
+      it("should return all deployments grouped by network", () => {
+        expect(
```

Comment:
are you only using expect for deep equals ? assert also has deep equals

Reply:
I'm using `expect` because I like it and it's more expressive than `assert` :-)